examples/spec/src/components/_pages/PageScreenHelmet.tsx

```diff
@@ -30,6 +33,8 @@ const PageScreenHelmet: React.FC = () => {
           appendLeft={appendLeft}
           appendRight={<div onClick={onRightClick}>{appendRight}</div>}
           noBorder={noBorder}
+          visible={visible}
+          preventBackSwipe={shouldPreventBackSwipe}
```

Comment:
음 그때 어셔가 피드백 주셨던거 생각해봤는데요 ㅎ 그래서

```suggestion
          preventSwipeBack={shouldPreventBackSwipe}
```

요 이름이 더 낫지 않나 하는 생각이 들었어요

Reply:
오.. 저는 그 때 4명 중 2인 이상이 preventBackSwipe 가 더 이해하기 쉽다고 해서 dx 가 이쪽이 더 나은가 보다 하고 바꿨는데, 토니는 어떤 면에서 더 낫다고 결론을 내린 거에요?

Reply:
더 표준어 같아서요 ㅋㅋ

Reply:
이번에 확실히 잘 정해야겠네요. 

swipe 라는 개념을 명시적으로 코드내에 표시한 것이 이 PR 이 처음이라서 이번에 결정한 거가 다음 작업 때도 계속 영향을 미칠테니.

Reply:
[06d6aed](https://github.com/daangn/karrotframe/pull/139/commits/06d6aedfcfd4c06ac44b5e3026b5869806b56e7e)

여기에서 `preventSwipeBack` 으로 변경했어요. 애플 공식 개발자 페이지에는 딱히 언급은 없는데 포럼 등에서 back swipe 보다는 swipe back 으로 더 자주 사용해서요.

Reply:
굿굿굿!